core/store/migrate/migrations/0155_remove_terra.sql

```diff
@@ -0,0 +1,54 @@
+-- +goose Up
+
+DROP TABLE terra_msgs;
+DROP TABLE terra_nodes;
+DROP TABLE terra_chains;
+
+-- +goose Down
+
+-- Table Definition ----------------------------------------------
+
+CREATE TABLE terra_chains (
```

Comment:
perhaps a naive question about db migrations: why are the tables being created after the drop if terra support is being removed completely? won't these created tables be forever after empty?

Reply:
Thats a rollback.

Reply:
ah, thanks